scripts/install/RegisterTestRunnerPlugins.php

```diff
@@ -439,6 +448,18 @@ class RegisterTestRunnerPlugins extends InstallAction
         ]
     ];
 
+    public static function getPlugin(string $pluginIdentifier): TestPlugin
```

Comment:
Since it can return null you need to update the signature :) 

```suggestion
    public static function getPlugin(string $pluginIdentifier): ?TestPlugin
```